### ISSUE TITLE
missing impls for LmdbRaw

### ISSUE THREAD

**Issue by MarkSwanson** [AltSysrq/lmdb-zero#4]
The docs for LmdbRaw state, "All integer and floating-point types have this trait".

However, this is not true. Almost no integer or floating-point types have this trait.

This breaks things like access.get::<str, f64>(&db, key)

My supported platforms are all x64, so I have no portability concerns.
It would be nice if the lmdb source had:

````
unsafe impl LmdbRaw for f64 { }
````

so I don't have to wrap things like this in slices.

**Comment by AltSysrq** [comment]
It looks like I forgot to update that part of the documentation; it was correct a while ago, but it was pretty easy to shoot yourself in the foot due to the alignment checks, so it was removed. (The statement now applies to the `LmdbRawIfUnaligned` trait.) The [`Unaligned`](https://docs.rs/lmdb-zero/0.3.1/lmdb_zero/struct.Unaligned.html) wrapper can be used to do direct copying. See also my comments on https://github.com/AltSysrq/lmdb-zero/issues/2 for an example.

While the x86 family is known for having lax alignment requirements, LLVM will still assume that a `&f64` has the proper alignment, and will readily emit SSE instructions that fault on misaligned pointers. Unfortunately I wasn't able to produce a simple example that does this, but the current alignment checking came about from me discovering that LMDB doesn't provide much in the way of alignment when I started getting `SIGBUS` due to a misaligned pointer. (And this was in fact on an amd64 system.)
